desk/app/activity.hoon

```diff
@@ -429,6 +447,7 @@
   |-
   ?~  sources  cor
   =.  cor  (refresh-summary i.sources)
+  =.  cor  (refresh-parent-summaries i.sources)
```

Comment:
Oh, do we expect the parent sources to not always exist in `indices`? Is that because of the way in which we do "less work" during migration logic?

Reply:
It's not that we don't expect them to exist, but that if a child source changes, we need to update the parent because we do some aggregation that should be affected by children.

Reply:
Hmm, makes me wonder then if `+refresh-summary` should recursively call itself for all the affected parent sources.

But, this should accomplish the same, just with superfluous passes, I think.

Reply:
yeah it probably should

Reply:
@Fang- am I good to merge as is though?